From 250d9f37a3f9143be848da6a8ec91436eaf2f522 Mon Sep 17 00:00:00 2001
From: Ding Yunliang <yunliang.ding@intel.com>
Date: Fri, 25 May 2018 10:46:59 +0800
Subject: [PATCH 1089/1240] media: intel-ipu4: imx274: update min exposure
 value

Message for Open Source:
This value was reported to CCA by hal as the exposure value,
so the minmum value should be 4 according to datasheet.

Message for Internal:
[Issue/Feature] Same as above.
[Root Cause/Changes] Same as above.

Change-Id: Ibb6007b78672619cb480f620baa39faf1b1c14ff
Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
---
 drivers/media/i2c/crlmodule/crl_imx274_configuration.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/crlmodule/crl_imx274_configuration.h b/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
index 3743d63..558faea 100644
--- a/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
@@ -1080,7 +1080,7 @@ static struct crl_v4l2_ctrl imx274_v4l2_ctrls[] = {
 		.ctrl_id = V4L2_CID_EXPOSURE,
 		.name = "V4L2_CID_EXPOSURE",
 		.type = CRL_V4L2_CTRL_TYPE_INTEGER,
-		.data.std_data.min = 8,
+		.data.std_data.min = 4,
 		.data.std_data.max = IMX274_MAX_SHS2,
 		.data.std_data.step = 1,
 		.data.std_data.def = 0x400,
-- 
2.7.4

